README.md

```diff
@@ -73,6 +73,8 @@ This contains the main ROS package. All your package-specific code goes here.
     Self explanatory
     * package.xml
     Self explanatory
+* .clang-format  
+This file contains the formatting settings to be used with clang-format. To use clang-format, please install it by running ```sudo apt-get install clang-format``` and running it with ```clang-format -style=file -i /path/to/file```. Make sure that this file is present on your workspace or home directory. It is also recommended that you install the clang-format extension for whatever editor you are using, to avoid running this command for each file. Also please note that clang doesn't currently support indentation after access modifiers (like public: and private:) so you have to indent the code after these yourself. Also doesn't support newline after : while using initialization lists in constructor.
```

Comment:
why use the extension?

Reply:
It is convenient, instead of writing commands for each file you can just press Ctrl-Shift-i for each file.

Reply:
When you will use extension how the extension will find the .clang file

Reply:
It just needs to either be in the home folder or the workspace folder that you have opened in vscode. I have clearly specified where configuration file should be, in the README.md